test/ecdsa_genpkey_sign_rawin.sh

```diff
@@ -5,18 +5,18 @@ set -eufx
 echo -n "abcde12345abcde12345" > testdata
 
 # generate private key as PEM
-openssl genpkey -provider tpm2 -algorithm EC -pkeyopt group:P-256 -out testkey.priv
+openssl genpkey -provider tpm2 -provider base -algorithm EC -pkeyopt group:P-256 -out testkey.priv
```

Comment:
I am sorry, my previous comment was incomplete: this `-provider base` feels superfluous too.

Reply:
Nothing to be sorry about.  I'm grateful for your attention for details! 😃 

Fixed